pubspec.yaml

```diff
@@ -13,12 +13,12 @@ dependencies:
     sdk: flutter
   flutter_web_plugins:
     sdk: flutter
-  js: ^0.6.4
+  js: ^0.7.1
```

Comment:
As I see from the [changelog](https://pub.dev/packages/js/changelog), it will change the minimum Dart SDK version (For Flutter it is 3.13+ I suppose).
Sounds like a major release and we need to mention it in the readme requirements section.

Reply:
Best solution will me to migrate to `dart:js_interop`, but I don't now how complicated it will be and not ready to spend our time at this moment.